fastcov.py

```diff
@@ -497,7 +497,10 @@ def exclProcessSource(fastcov_sources, source, exclude_branches_sw, include_bran
                     line_to_func[l] = set()
                 line_to_func[l].add(f)
 
-            if "LCOV_EXCL_LINE" in line:
+            if "LCOV_EXCL_LINE" not in exclude_line_marker:
+                exclude_line_marker.append("LCOV_EXCL_LINE") # add default value to list
```

Comment:
Should we just make this the argparse default? i.e.:
```
parser.add_argument('-ce', '--custom-exclusion-marker', dest='exclude_line_marker', nargs="+", metavar='', default=["LCOV_EXCL_LINE"], help='Filter: Add filter for lines that will be excluded from coverage (same behavior as "LCOV_EXCL_LINE)')
```

Reply:
If I am not mistaken, adding this as a default would spoil the early outs we have in L468 and L489 as we also have to check for the other LCOV exclusion options (LCOV_EXCL_START/STOP/BR_LINE).